test/ui-test/src/screens/StatusWalletScreen.py

```diff
@@ -350,7 +347,9 @@ def verify_saved_address_doesnt_exist(self, name: str):
                 verify_failure(f'FAIL: saved address {name} exists')
 
     def verify_transaction(self):
-        print("TODO: fix notification and ensure there is one")
+        ephemeral_notification_list = get_obj(MainWalletScreen.EPHEMERAL_NOTIFICATION_LIST.value)
+        squish.waitFor("ephemeral_notification_list.count > 0", 60*1000)
+        verify(str(ephemeral_notification_list.itemAtIndex(0).primaryText ) == "Transaction pending...", "Tx was not sent!")
```

Comment:
What will happen with this comparison if we want to run the tests in another language? :P

Could we use instead the `enabled` property??